pkg/util/humanizeutil/humanize.go

```diff
@@ -113,3 +114,14 @@ func (b *BytesValue) String() string {
 func (b *BytesValue) IsSet() bool {
 	return b.isSet
 }
+
+// DataRate formats the passed byte count over duration as "x MiB/s".
+func DataRate(bytes int64, elapsed time.Duration) string {
+	if bytes == 0 {
+		return "0"
+	}
+	if elapsed == 0 {
+		return "inf"
+	}
+	return fmt.Sprintf("%0.2f MiB/s", (float64(bytes)/elapsed.Seconds())/float64(1<<20))
```

Comment:
One could imagine using `humanizeutil.IBytes` as `%s/s` to arrive at something like `1.4 GiB/s` once things got faster, but I'm happy hardcoding `MiB/s`

Reply:
Oh, yes, please do this now.

Reply:
So I actually started with IBytes but I didn't actually like it switching between KiB and MiB -- it made it a little harder to compare across runs than always using a fixed unit.

Reply:
Fair point. I'm happy to try it like this and switch to autoscaling units later if it ends up being the wrong tradeoff.